src/core/scenario.jl

```diff
@@ -254,7 +254,16 @@ function Scenario(d::Dict; flex_hvac_from_json=false)
 
     max_cooling_demand_kw = 0
     if haskey(d, "CoolingLoad") && !haskey(d, "FlexibleHVAC")
-        add_doe_reference_names_from_elec_to_thermal_loads(d["ElectricLoad"], d["CoolingLoad"])
+        assign_elec_doe_reference_name = true
+        cooling_keys_exceptions = ["annual_fraction", "monthly_fraction", "loads_fraction"]
```

Comment:
are all of these fractions the fraction of the electric load that is assumed to be cooling load? it would be helpful to put it in the name if so, maybe append `_of_electric_load` and make the monthly and per time step values plural. For example:
```
annual_fraction_of_electric_load
monthly_fractions_of_electric_load
per_time_step_fractions_of_electric_load
```
We should also document how these are used in the `CoolingLoad` doc string. As a user, for example, I would want to know if the cooling load fraction is removed from the electric load or not.

Reply:
I like that, so I changed it. I'll update the doc string to clarify.

Reply:
It looks like cooling electric is **not** currently subtracted off of the total electric load, so I'll work on fixing that.

Reply:
I addressed the naming, doc string, and subtracting the cooling electric in the load balance constraints in the last two commits. Waiting to see if I broke anything from the tests.